### ISSUE TITLE
Schematic improvement: Tie VBUS and VSYS together on each Pico

### ISSUE THREAD

**Issue by kpfleming** [hrvach/deskhop#62]
According to the section 4.5 of the Pico datasheet, when only a single power supply is used for the Pico VBUS and VSYS can be tied together. This eliminates a small voltage drop across a diode in the power circuit, improving efficiency and eliminating some voltage ripple on VSYS.

**Comment by s51517765** [comment]
Is this same as the one below?

https://github.com/hrvach/deskhop/issues/9#issuecomment-1874975203

**Comment by kpfleming** [comment]
No, that's for powering both Picos from one source. This one is for a small improvement on each side, but that does not tie the sides together.
